app/src/main/java/org/p2p/wallet/receive/list/TokenListFragment.kt

```diff
@@ -75,6 +75,18 @@ class TokenListFragment :
                     recyclerView.smoothScrollToPosition(0)
                 }
             }
+            recyclerView.isVisible = items.isNotEmpty()
+            emptyView.isVisible = items.isEmpty()
+        }
+    }
+
+    override fun showEmpty(searchText: String) {
+        with(binding) {
+            val text = getString(R.string.receive_token_list_not_found, searchText)
```

Comment:
yeah, I know that getString returns `text` :) 
what im interested in when reading this code - what kind of text? 
`notFoundText` `notFoundTitle` is good enough IMHO